test/ExecuteUpdateTest.cpp

```diff
@@ -126,6 +122,27 @@ TEST(ExecuteUpdate, executeUpdate) {
       NumTriples(0, 1, 1));
   expectExecuteUpdate("INSERT DATA { <a> <b> <c> }; DELETE WHERE { ?s ?p ?o }",
                       NumTriples(0, 9, 9));
+  expectExecuteUpdate("CLEAR SILENT GRAPH <x>", NumTriples(0, 0, 0));
+  expectExecuteUpdate("CLEAR DEFAULT", NumTriples(0, 8, 8));
+  expectExecuteUpdate("CLEAR SILENT NAMED", NumTriples(0, 0, 0));
+  expectExecuteUpdate("CLEAR ALL", NumTriples(0, 8, 8));
+  expectExecuteUpdate("DROP GRAPH <x>", NumTriples(0, 0, 0));
+  expectExecuteUpdate("DROP SILENT DEFAULT", NumTriples(0, 8, 8));
+  expectExecuteUpdate("DROP NAMED", NumTriples(0, 0, 0));
+  expectExecuteUpdate("DROP SILENT ALL", NumTriples(0, 8, 8));
+  expectExecuteUpdate("ADD <x> TO <x>", NumTriples(0, 0, 0));
+  expectExecuteUpdate("ADD SILENT <x> TO DEFAULT", NumTriples(0, 0, 0));
+  expectExecuteUpdate("ADD DEFAULT TO <x>", NumTriples(8, 0, 8));
+  expectExecuteUpdate("ADD SILENT DEFAULT TO DEFAULT", NumTriples(0, 0, 0));
+  expectExecuteUpdate("MOVE SILENT DEFAULT TO DEFAULT", NumTriples(0, 0, 0));
+  expectExecuteUpdate("MOVE GRAPH <x> TO <x>", NumTriples(0, 0, 0));
+  expectExecuteUpdate("MOVE <x> TO DEFAULT", NumTriples(0, 8, 8));
+  expectExecuteUpdate("MOVE DEFAULT TO GRAPH <x>", NumTriples(8, 8, 16));
+  expectExecuteUpdate("COPY DEFAULT TO <x>", NumTriples(8, 0, 8));
+  expectExecuteUpdate("COPY DEFAULT TO DEFAULT", NumTriples(0, 0, 0));
+  expectExecuteUpdate("COPY <x> TO DEFAULT", NumTriples(0, 8, 8));
+  expectExecuteUpdate("CREATE SILENT GRAPH <x>", NumTriples(0, 0, 0));
+  expectExecuteUpdate("CREATE GRAPH <y>", NumTriples(0, 0, 0));
```

Comment:
It looks like for the setup of these tests the named graphs are all empty. Is this easilty fixable, s.t. the tests with the NAMED keyword don't always have `(0, 0, 0)` as the result?

Reply:
Added another group of tests where we have some named graphs.